components/gallery/useGalleryItem.ts

```diff
@@ -34,17 +37,26 @@ export const useGalleryItem = () => {
   // const { id: collectionID, item: id } = tokenIdToRoute(params.id)
 
   const { urlPrefix } = usePrefix()
-  const { data } = useGraphql({
+  const { data, refetch } = useGraphql({
     queryName: urlPrefix.value === 'rmrk' ? 'nftByIdWithoutRoyalty' : 'nftById',
     variables: {
       id: params.id,
     },
+    options: {
+      fetchPolicy: 'network-only',
+    },
+  })
+  useSubscriptionGraphql({
+    query: `   nftEntities(where: {id_eq: "${params.id}"}) {
```

Comment:
Why there is a query string? Please use graphqlfile instead as written in the task desc

Reply:
Anyway the query is not correct, please check `nftByIdMinimal`

Reply:
Thanks. I have updated the query.
Now the query string is composed of querys from `queries/rmrk/subsquid/nftByIdMinimal.graphql` and `queries/nftByIdMinimal.graphql`

Reply:
Still this is not correct

You 
```graphql
nftEntities(where: {id_eq: "${params.id}"}) {
```

nftByIdMinimal
```graphql
nft: nftEntityById(id: $id) {
```